src/PresentationalComponents/Modal/ModalContent.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import classNames from 'classnames';
+
+/**
+ * This is a dumb component that only recieves properties from a smart component.
+ * Dumb components are usually functions and not classes.
+ *
+ * @param props the props given by the smart component.
+ */
+
+const ModalContent = ({ app, variant, className, ...props }) => {
+
+    const modalClasses = classNames (
+        'ins-c-modal__content',
+        className
+    );
+
+    switch (variant) {
```

Comment:
This can be simplified with object literals
```JSX
return {
  notEntitled: <p className ={ modalClasses } { ...props }> { app } is not entitled </p>,
  notSetUp: <p className ={ modalClasses } { ...props }> { app } has not been set up </p>
}[variant]
```

Reply:
Fixed!